tests/Integration/Behaviour/Features/Scenario/Cart/Calculation/Currency/currency.feature

```diff
@@ -90,7 +90,7 @@ Feature: Cart calculation with currencies
     Given currency "currency2" is the current one
     Given there is a product in the catalog named "product1" with a price of 19.812 and 1000 items in stock
     When I add 3 items of product "product1" in my cart
-    Then my cart total should be 83.0 tax included
+    Then my cart total should be 83.1 tax included
```

Comment:
Dont' think there's a problem here? 😅
83.1 instead of 83.0 with the previous calculation method
I know it's supposed to be fixed in another PR but still merging invalid tests seems a bit risky

Reply:
yep, that's why I would like this PR not to be merged too soon, we should wait for the FIX of this issue first